package.json

```diff
@@ -38,12 +39,9 @@
         "typechain": "^7.0.1"
     },
     "dependencies": {
+        "@openzeppelin/contracts": "^4.5.0",
         "@typechain/ethers-v5": "^9.0.0",
         "@typechain/hardhat": "^5.0.0",
-        "@unirep/circuits": "git+https://github.com/Unirep/circuits.git",
```

Comment:
We should keep these dependencies but point them to a version number. Then the packages can be linked locally for development if needed.